calico_versioned_docs/version-3.25/network-policy/get-started/calico-policy/calico-network-policy.mdx

```diff
@@ -217,7 +217,7 @@ spec:
 
 ### Apply network policies in specific order
 
-To control the order/sequence of applying network policies, you can use the **order** field (with precedence from the lowest value to highest). Defining policy **order** is important when you include both **action: allow** and **action: deny** rules that may apply to the same endpoint.
+To control the order/sequence of applying network policies, you can use the **order** field (with precedence from the lowest value to highest). Defining policy **order** is important when you include both **action: allow** and **action: deny** rules that may apply to the same endpoint. Because Kubernetes NetworkPolicies don't have an explicit order field, they are treated as though they had an implicit order of 1000.
```

Comment:
```suggestion
To control the order/sequence of applying network policies, you can use the **order** field (with precedence from the lowest value to highest). Defining policy **order** is important when you include both **action: allow** and **action: deny** rules that may apply to the same endpoint. Because Kubernetes network policies don't have an explicit order field, they are treated as though they had an implicit order of 1000.
```